packtools/sps/validation/formula.py

```diff
@@ -134,7 +134,7 @@ def validate_label(self):
             is_valid=is_valid,
             expected="label",
             obtained=label,
-            advice='Mark each formula label with <label> inside <disp-formula>. Consult SPS documentation for more detail.',
+            advice='Mark each label with <label> inside <disp-formula>. Consult SPS documentation for more detail.',
```

Comment:
@Rossi-Luciano adicion `id="{item_id}"` na mensagem. Lembre-se de instanciar `item_id`

Troque `Mark each label with <label> inside <disp-formula>. Consult SPS documentation for more detail.` por
`Mark each label with <label> inside <disp-formula id="{item_id}">. Consult SPS documentation for more detail.`

Reply:
OK, @robertatakenaka